### ISSUE TITLE
TapToPlace Small Issue

### ISSUE THREAD

**Issue by HaraldHeide** [Microsoft/HoloToolkit#85]
Have a feeling that we should check if the anchorManager.AnchorStore exists and are ready before Attach or Remove Anchors in the OnInputClicked handler.

Something like:
                if (anchorManager.AnchorStore != null)
                {
                    // Add world anchor when object placement is done.
                    anchorManager.AttachAnchor(gameObject, SavedAnchorFriendlyName);
                }

**Comment by StephenHodgson** [comment]
That is correct. This issue belongs in the [HTK-Unity Project](https://github.com/Microsoft/HoloToolkit-Unity).
